sci-biology/fastqc/fastqc-0.11.3.ebuild

```diff
@@ -1,39 +1,45 @@
-# Copyright 1999-2015 Gentoo Foundation
+# Copyright 1999-2020 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=5
+EAPI=7
 
 inherit java-pkg-2 eutils java-ant-2
 
 DESCRIPTION="Quality control FASTA/FASTQ sequence files"
-HOMEPAGE="http://www.bioinformatics.babraham.ac.uk/projects/fastqc/"
-SRC_URI="http://www.bioinformatics.babraham.ac.uk/projects/fastqc/fastqc_v"${PV}"_source.zip"
+HOMEPAGE="https://www.bioinformatics.babraham.ac.uk/projects/fastqc/"
+SRC_URI="https://www.bioinformatics.babraham.ac.uk/projects/fastqc/fastqc_v"${PV}"_source.zip"
 
 LICENSE="GPL-3+"
 SLOT="0"
-KEYWORDS=""
-IUSE=""
+KEYWORDS="~amd64"
 
-DEPEND="sci-biology/picard
-	sci-libs/jhdf5
-	>=virtual/jre-1.5:*"
-RDEPEND="${DEPEND}
+RDEPEND="
 	dev-lang/perl
+	>=virtual/jre-1.5:*
+	"
+DEPEND="$RDEPEND
 	>=virtual/jdk-1.5:*
-	dev-java/ant-core"
+	"
 
 S="${WORKDIR}"/FastQC
 
 src_prepare(){
 	cp "${FILESDIR}"/build.xml . || die
+	eapply_user
 }
 
 src_compile(){
 	ant || die
 }
 
 src_install(){
-	dobin fastqc run_fastqc.bat
+	insinto "opt/${PN}"
+	doins -r bin
+	chmod a+x "${ED}/opt/${PN}/bin/fastqc"
+	# Add the package's bin directory to the PATH.
+	echo "PATH=\"${EPREFIX}/opt/${PN}/bin\"" > "${T}/00fastqc" || die
```

Comment:
better to create this file in `files/00fastqc` instead of doing an `echo`.
Don't need the `${EPREFIX}`.

Reply:
Without the ${EPREFIX} it would not work in a Prefix environment, would it?

Reply:
`env-update` handles the creating of `$PATH` from `env.d` files, it managed the `${EPREFIX}` too.

Reply:
From my experiments: `env-update` is called automatically, but does not manage `${EPREFIX}`. Instead, one needs to call `hprefixify` defined by the `prefix` eclass. Example: [baselayout-2.7.ebuild, line 187](https://github.com/gentoo/gentoo/blob/e0ec6727d673a6ac44c7ee7d82d2ebc609e1f511/sys-apps/baselayout/baselayout-2.7.ebuild#L187)